### PR TITLE
Story 1 display all invoices

### PR DESCRIPTION
**Pull request by henryppercy**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,70 +1,39 @@
-# Getting Started with Create React App
+# 2022 August - Hawk's Cohort - React Invoice App
+
+## 1. About
+### Content
+Project 4 relating to module 5 of iO Full-Stack Web Development.
+
+### Objective
+Group project with the objective of build an Invoice Management Application to test knowledge acquired
```
Comment (nealioa) [inline README.md]: Maybe "building" instead of "build"?

File: README.md [hunk 2]
```diff
@@ -1,70 +1,39 @@
...
+### Content
+Project 4 relating to module 5 of iO Full-Stack Web Development.
+
+### Objective
+Group project with the objective of build an Invoice Management Application to test knowledge acquired 
+during module 5 of iO Academy Full-Stack curriculum:
++ Agile work environment best practices in a SCRUM methodology
++ Git / Github workflow and best practices as a developer team
++ Component Driven Development using the React.js Library
++ CSS frameworks  using Bootstrap 5.2.2
```
Comment (nealioa) [inline README.md]: Delete one of the spaces between "frameworks" and "using".

File: README.md [hunk 3]
```diff
@@ -1,70 +1,39 @@
...
+### Technologies
++ React.js - https://reactjs.org/
++ Bootstrap - https://getbootstrap.com/
++ Git - https://github.com/com
+
+## 2. Installation
+Follow the next steps to install locally:
++ ### Install Git
+  Using the following the instructions for the corresponding OS - https://github.com/git-guides/install-git
++ ### Installing the Data Base
```
Comment (nealioa) [inline README.md]: Go with "Database" - most people are ok with the one word.

File: src/App.js [hunk 4]
```diff
@@ -1,24 +1,12 @@
-import logo from './logo.svg';
-import './App.css';
+import 'bootstrap/dist/css/bootstrap.min.css';
+import 'bootstrap/dist/js/bootstrap.bundle.min';
+import Invoices from './components/Invoices';
+import 'bootstrap/dist/css/bootstrap.min.css';
```
Comment (nealioa) [inline src/App.js]: Why are these repeated?

File: src/components/Invoices/InvoiceCards/index.js [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+            month: 'long',
+            day: 'numeric',
+        };
+        return (new Intl.DateTimeFormat('en-GB', options).format(date));
+    }
+
+    return (
+        <>
+            <div className="bg-white d-flex flex-wrap mb-2 justify-content-between align-items-center p-3 bd-highlight border rounded" key={id}>
+                    <div className="pe-4">#<strong>{invoice_id}</strong></div>
```
Comment (nealioa) [inline src/components/Invoices/InvoiceCards/index.js]: You can outdent this and the next four to line up with line 42.

File: src/components/Invoices/index.js [hunk 6]
```diff
@@ -0,0 +1,99 @@
...
+    const fetchInvoices = async () => {
+        const response = await fetch('http://localhost:8080/invoices');
+        if (!response.ok) {
+            throw new Error();
+        }
+        return await extractResponseData(response);
+    }
+
+    // --- on mount logs the data retrieved ---
+    // --- Replace console.log(invoiceData) with setState inside the .then() to display data ---
```
Comment (nealioa) [inline src/components/Invoices/index.js]: Looks like this has been done, so the comment is redundant now. Remove it.

File: src/components/Invoices/index.js [hunk 7]
```diff
@@ -0,0 +1,99 @@
...
+            });
+    }, []);
+
+    return (
+        <>
+            <div className="py-5 px-3">
+            <header className="d-flex justify-content-between align-items-end flex-wrap pt-3 px-0">
+                <div>
+                    <h1 className="fw-bolder">Invoices</h1>
+                    <p>There are 7 unpaid invoices</p>
```
Comment (nealioa) [inline src/components/Invoices/index.js]: The number of invoices should be calculated.

**Review by nealioa — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (PNaing107) [reply README.md]: done

File: README.md (hunk 2 above)
Comment (PNaing107) [reply README.md]: done

File: README.md (hunk 3 above)
Comment (PNaing107) [reply README.md]: done

File: src/App.js (hunk 4 above)
Comment (PNaing107) [reply src/App.js]: removed line 4 & 5 (duplicate import statements)

File: src/components/Invoices/InvoiceCards/index.js (hunk 5 above)
Comment (PNaing107) [reply src/components/Invoices/InvoiceCards/index.js]: done

File: src/components/Invoices/index.js (hunk 6 above)
Comment (PNaing107) [reply src/components/Invoices/index.js]: done

File: src/components/Invoices/index.js (hunk 7 above)
Comment (PNaing107) [reply src/components/Invoices/index.js]: done

File: src/helpers/utils.js [hunk 8]
```diff
@@ -0,0 +1,31 @@
...
+    )
+}
+
+export const currencyFormatter = (invoice_total) => {
+    const options = {
+        style: 'currency',
+        currency: 'GBP',
+    };
+    const invoicesTotalFloat = parseFloat(invoice_total);
+    return invoicesTotalFloat.toLocaleString("en-GB", options);
```
Comment (nealioa) [inline src/helpers/utils.js]: Add space after currency symbol.

**Review by nealioa — CHANGES_REQUESTED** [review]

File: src/helpers/utils.js (hunk 8 above)
Comment (PNaing107) [reply src/helpers/utils.js]: done

**Review by nealioa — APPROVED** [review]

**Pull request merged**